extensions/amp-consent/integrating-consent.md

```diff
@@ -0,0 +1,131 @@
+# Integrate your Consent Management Platform (CMP) with AMP Consent
+
+## Overview
+The feature is still in experimental mode. Please opt-in using the `amp-consent-v2` flag. To opt-in, you can either run `AMP.toggleExperiment('amp-consent-v2')` in console, or turn on the experiment from the [experiments managing page](https://cdn.ampproject.org/experiments.html).
+
+If you operate a CMP to help publishers to manage their user consents, you may want to integrate your service in `amp-consent`. This will enable your customers to use your service in AMP HTML pages.
+
+## Before you begin
+
+To add your consent management service to AMP runtime, it is expected that you do:
+* Set the remote endpoint to handle requests from AMP. This is for the purpose of determining if the user consent will be required.
+* Make sure the prompted consent collecting page communicates well with the AMP runtime using the provided APIs.
+* Check in your consent configuration to the AMP code base.
+* Be aware of the restrictions that the AMP runtime applies to ensure a good user experience. These includes
+    * Enforce the size of the consent prompt. The only two allowed sizes are the initial size (`width: 100vw`, `height: 30vh`), and the fullScreen (`width: 100vw`, `height: 100%`) after user interactions.
```

Comment:
I changed it to full size screen. It's not an attribute, I am trying to describe the two type of allowed sizes here. 
Consent prompt can move between these two sizes, but the size change is in one directly. It's the initial size by default, CMP call our full screen API to expand the prompt to full size. After that the prompt can only be closed, it can not go back to the initial smaller size.